src/js/resize.js

```diff
@@ -0,0 +1,22 @@
+import Split from 'split.js'
+
+const PAGE_WIDTH = 960
+
+const isSplited = () => document.getElementsByClassName('gutter').length > 0
+const isBigScreen = () => window.innerWidth > PAGE_WIDTH
+const main = document.getElementById('main')
+const gutter = () => document.getElementsByClassName('gutter')[0]
+
+const makeGutter = () => {Split(['#left_pane', '#right_pane'], {
+    gutterSize: 8,
```

Comment:
Btw there is option `gutter` which should be a function like that:
```js
(index, direction) => {
    const gutter = document.createElement('div')
    gutter.className = `gutter gutter-${direction}`
    return gutter
}
```

We could return our own screen divider instead of letting split.js to make own gutter.